src/main/java/net/mcreator/workspace/WorkspaceFolderManager.java

```diff
@@ -105,6 +105,11 @@ public List<String> getStructureList() {
 		return structures;
 	}
 
+	public File getVillagerProfessionTextureFileForName(String villagerProfessionTextureName) {

```

Comment:
No matter which texture type you use, this method should not be added. You should use the new general method.